contracts/UniswapV2Factory.sol

```diff
@@ -46,4 +47,15 @@ contract UniswapV2Factory is IUniswapV2Factory {
         require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
         feeToSetter = _feeToSetter;
     }
+    
+    function setProtocolFee(uint8 _protocolFee) external {
+        require(msg.sender == feeToSetter, 'UniswapV2: FORBIDDEN');
+        require(_protocolFee > 0, 'UniswapV2: FORBIDDEN_FEE');
```

Comment:
I think we should set this to `require(_protocolFee >= 5)` to have a maximum protocol fee of 0.05%

Reply:
This can later be changed by the DXdao, we can enforce and add more rules with a contract between the DXdao and TokenPairFactory, where it will add more rules for limits and fee distribution.
Not adding the change now since that conditions can be added later and we want to keep the changes minimal.